### PR TITLE
Add new recipe for vegan brownies to Quarto

### LINKED ISSUE #39: Add vegan Brownie recipe
Here's a delicious and easy vegan brownie recipe for you to try:
Vegan Brownie Recipe

Ingredients:

    1 cup all-purpose flour
    1 cup granulated sugar
    1/2 cup unsweetened cocoa powder
    1 teaspoon baking powder
    1/2 teaspoon salt
    1/2 cup water
    1/2 cup vegetable oil or melted coconut oil
    1 teaspoon vanilla extract
    1/2 cup vegan chocolate chips (optional)
    1/4 cup chopped nuts (optional)

Instructions:

    Preheat Oven:
    Preheat your oven to 350°F (175°C). Lightly grease an 8x8-inch baking pan or line it with parchment paper.

    Mix Dry Ingredients:
    In a large mixing bowl, whisk together the flour, sugar, cocoa powder, baking powder, and salt until well combined.

    Add Wet Ingredients:
    Add the water, vegetable oil (or melted coconut oil), and vanilla extract to the dry mixture. Stir until the batter is smooth and well combined.

    Fold in Add-ins:
    If using, fold in the vegan chocolate chips and chopped nuts.

    Bake:
    Pour the batter into the prepared baking pan, spreading it out evenly. Bake for 25-30 minutes, or until a toothpick inserted into the center comes out clean or with a few moist crumbs.

    Cool:
    Allow the brownies to cool completely in the pan on a wire rack before cutting into squares.

    Serve:
    Enjoy your vegan brownies plain or with a scoop of vegan ice cream on top!

Tips:

    For an extra fudgy texture, reduce the baking time slightly.
    Feel free to customize your brownies with your favorite mix-ins like dried fruits or seeds.

Enjoy your homemade vegan brownies!

### PR DESCRIPTION
**Pull request by MariaJakubowski**
Fixes #39

### CODE REVIEW

File: recipes/vegan_brownies.qmd [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Vegan Brownies"
+image: "https://img.taste.com.au/5hsR3Ore/w720-h480-cfill-q80/taste/2016/11/classic-chewy-brownie-102727-1.jpeg"
+image-alt: "https://img.taste.com.au/5hsR3Ore/w720-h480-cfill-q80/taste/2016/11/classic-chewy-brownie-102727-1.jpeg"
+categories:
+  - Vegan
```
Comment (lnnrtwttkhn) [inline recipes/vegan_brownies.qmd:6]: Please add your author information to the YAML header:

```yaml
author: <your name here>
```

**Review by lnnrtwttkhn — CHANGES_REQUESTED** [review]
Thanks for this PR @MariaJakubowski! Looks great! I only have a minor request (adding your author information). Thank you!

**Comment by lnnrtwttkhn** [comment]
- [ ] @MariaJakubowski could you also please reference the issue that this PR is fixing in the description above? thanks!

File: recipes/vegan_brownies.qmd (hunk 1 above)
Comment (MariaJakubowski) [reply recipes/vegan_brownies.qmd:6]: I added my name to the Quarto file.

**Comment by lnnrtwttkhn** [comment]
@allcontributors please add @MariaJakubowski for bug, content, code

**Comment by lnnrtwttkhn** [comment]
Looks great, thanks for the additional changes @MariaJakubowski. Ready to merge!

**Pull request merged**